backend/tests/Functional/Controller/GlobalTagVisibilityTest.php

```
@@ -0,0 +1,79 @@
+        $defaultLibrary = $this->getJson('/api/comics');
+        self::assertResponseIsSuccessful();
+        self::assertSame(['Visible'], array_column($defaultLibrary['comics'], 'title'));
+
+        $ordinaryFilter = $this->getJson('/api/comics?tags=Marvel');
+        self::assertResponseIsSuccessful();
+        self::assertSame(['Visible'], array_column($ordinaryFilter['comics'], 'title'));
+
+        $hiddenFilter = $this->getJson('/api/comics?tags=Hidden');
+        self::assertResponseIsSuccessful();
+        self::assertSame(['Private'], array_column($hiddenFilter['comics'], 'title'));
+        self::assertTrue($hiddenFilter['comics'][0]['tags'][1]['hideFromLibrary']);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Functional\Controller\GlobalTagVisibilityTest::assertTrue().


(staticMethod.notFound)